service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Component
+@ConfigurationProperties(prefix = "gp-connect")
```

Comment:
I'm surprised this prefix works. Javadoc says "A valid prefix is defined by one or more words separated with dots" and the actual value is "gpConnect". Maybe change this to "gpConnect" just to be safe.